ansible/roles/desktop/tasks/main.yml

```diff
@@ -1,7 +1,6 @@
 ---
-  - name: install gnome3 packages
+  - name: install packages
     apt:
       name: "{{ item }}"
     with_items:
-      - gnome-session
-      - gdm3
```

Comment:
Do we not need GDM3/LightDM?

Reply:
Nope. Lightdm is installed automatically. xubuntu-core just includes all the DE and theme and stuff to get it going without applications like libreoffice and minesweeper etc

Reply:
We could most definitely tune it down to the bare minimum if we wanted to save some space